src/styles/blocks.scss/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $color-secondary;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &-value {
+      color: $color-main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-label {
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $color-main-accent;
+    }
+  }
+
+  &__buy {
+    font-weight: 700;
+    font-size: 14px;
+    color: #fff;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $color-blue-accent;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
```

Comment:
It is better to set box-sizing for all elements using * selector to make sure that all elements will have the same behaviour